Moving catchcn CO2 file to bcs dir

biljanaorescanin opened the pull request:
Addressing issue #471.  
This PR is using CO2 file needed for CatchCN offline simulations from boundary conditions area. 

To accommodate for this in "legacy_bcs" area we added this file to these boundary conditions versions. 
`Icarus-NLv3;
Icarus ;
Ganymed-4_0`

@gmao-jkolassa do you need me to add this to any other boundary conditions version? 
I will add one more PR to GEOSldas to change path for ldas_setup as well.

gmao-jkolassa commented:
Thanks @biljanaorescanin ! Could you please also add this file to the `Heracles-NL` boundary conditions version, as that is what some of the Catchment-CN4.0 simulations are using. Thanks!

biljanaorescanin commented:
@gmao-jkolassa I just added `Heracles-NL` its good I've asked I didn't think anyone uses that set.  

@mathomp4 My understanding is that NAS sync's to DISCOVER, is that correct? If it is than this should be there with first rsync that happens? Since we already moved BCS files to be used from `legacy_bcs` path and not `ltakacs` area.

mathomp4 commented:
> @mathomp4 My understanding is that NAS sync's to DISCOVER, is that correct? If it is than this should be there with first rsync that happens? Since we already moved BCS files to be used from `legacy_bcs` path and not `ltakacs` area.

It's the other way. We sync discover to NAS. So, if this is in whatever BCSDIR is on discover, we should get it at NAS. At the moment, we probably don't have every BCs directory at NAS because we are in a space-crunch. But if things fail, we can figure it out.
